src/Uno.Toolkit.UI/Behaviors/StatusBar.iOS.cs

```diff
@@ -0,0 +1,33 @@
+﻿#if __IOS__
+using System;
+using System.Collections.Generic;
+using System.Text;
+using UIKit;
+
+using XamlColor = Windows.UI.Color;
+using XamlStatusBar = Windows.UI.ViewManagement.StatusBar;
+
+namespace Uno.Toolkit.UI
+{
+	public static partial class StatusBar
+	{
+		static partial void SetForegroundCore(XamlColor value)
+		{
+			XamlStatusBar.GetForCurrentView().ForegroundColor = value;
+		}
+
+		static partial void SetBackgroundCore(XamlColor value)
+		{
+			const int StatusBarViewTag = 38482;
```

Comment:
Why this number? Make sure to document.

Reply:
There isnt actually much information behind this number. It is been shown used in:
https://stackoverflow.com/questions/56651245/how-to-change-the-status-bar-background-color-and-text-color-on-ios-13
https://github.com/maxkoshevoi/XamarinCommunityToolkit/blob/main/src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/StatusBar/PlatformStatusBarEffect.ios.cs#L48
Not sure if this is a known constant value or chosen via dice roll...

Reply:
`ViewWithTag` is a miss on the first run. Another constant [38482458385](https://www.google.com/search?q=38482458385) seem to have more results on the internet, but couldnt be used here as it overflows `nint`... There is still no explanation on that either.
The value _appears to be_ just an arbitrary number used to avoid recreating the same view.